### PR TITLE
op-wheel: add cheat code command

### PR DESCRIPTION
**Pull request by tynes**
**Description**

Allows it to set into the state arbitrary code at an arbitrary account. This will make it very easy to patch contracts that contain immutables. Then we can patch the contracts to have different immutables without needing to update the proxies and worry about that.

We can also cheat the impl of the multisig to be a call forwarder so that we do not need to cheat so many storage slots and forget about particular slots.

### CODE REVIEW

File: op-wheel/cheat/cheat.go [hunk 1]
```diff
@@ -379,6 +379,13 @@ func SetBalance(addr common.Address, amount *big.Int) HeadFn {
 	}
 }
 
+func SetCode(addr common.Address, code string) HeadFn {
+	return func(headState *state.StateDB) error {
+		headState.SetCode(addr, common.FromHex(code))
```
Comment (protolambda) [inline op-wheel/cheat/cheat.go:384]: it should `code []byte`, otherwise it's going to panic on invalid strings

File: op-wheel/commands.go [hunk 2]
```diff
@@ -271,6 +271,17 @@ var (
...
 	}
+	CheatSetCodeCmd = cli.Command{
+		Name: "code",
+		Flags: []cli.Flag{
+			DataDirFlag,
+			addrFlag("address", "Address to change code of"),
+			cli.StringFlag{Name: "code", Usage: "New code of the account"},
+		},
+		Action: CheatAction(false, func(ctx *cli.Context, ch *cheat.Cheater) error {
+			return ch.RunAndClose(cheat.SetCode(addrFlagValue("address", ctx), ctx.String("code")))
```
Comment (protolambda) [inline op-wheel/commands.go:282]: Similar to `addrFlag` helper, we should define a `hexutils.Bytes` generic-flag creation helper, to make bytes-flags easy and safe (errors are handled as part of flag-parsing then, and don't cause panics).

**Review by protolambda — COMMENTED** [review]
It works, but I'd strongly suggest using a more safe approach to flag parsing, so we don't get panics on bad inputs. See comments on how to resolve.

**Comment by tynes** [comment]
Updated based on PR comments

**Review by protolambda — APPROVED** [review]

**Pull request merged**